sysutils/clamav-server/files/patch-etc-freshclam-conf-macports.diff

```diff
@@ -18,25 +18,25 @@
  
  # Maximum size of the log file.
  # Value of 0 disables the limit.
-@@ -48,7 +48,7 @@
- 
- # This option allows you to save the process identifier of the daemon
+@@ -52,7 +52,7 @@
+ # It is recommended that the directory where this file is stored is
+ # also owned by root to keep other users from tampering with it.
  # Default: disabled
--#PidFile /var/run/freshclam.pid
+-#PidFile /run/clamav/freshclam.pid
 +PidFile @PREFIX@/var/run/clamav/freshclam.pid
```

Comment:
Why would one put this into /opt/local?

Reply:
Because `${prefix}/var` is the place MacPorts uses for files created by MacPorts installs. There are dozens of ports that do this, including `clam', the port on which this port is based.

A few of them: `clamav`, `dcc`, `elasticsearch`, `macports`, `mailman2`, `named`, `privoxy`, `pwcheck`, `solr`,  `squid`, `apache2`, `dbus`, `dcc`, `dovecot`, `lighttpd`, `logrotate`, `named`, `nginx`, `redis`, `rspamd`, `solr`, `squid`, `tor`.